### ISSUE TITLE
Panic in (*Session).Send()

### ISSUE THREAD

**Issue by palkan** [anycable/anycable-go#34]
```
panic: send on closed channel 
goroutine 19 [running]: 
github.com/anycable/anycable-go/node.(*Session).Send(0x1a3f5e00, 0x1b99c000, 0x6e81e, 0xa5bdd) 
/Users/palkan/dev/golang/src/github.com/anycable/anycable-go/node/session.go:149 +0x48 
github.com/anycable/anycable-go/node.(*Hub).broadcastToStream(0x19ba22a0, 0x1b1f0bf0, 0xe, 0x1b804000, 0x6e7d4) 
/Users/palkan/dev/golang/src/github.com/anycable/anycable-go/node/hub.go:248 +0x2a1 
github.com/anycable/anycable-go/node.(*Hub).Run(0x19ba22a0) 
/Users/palkan/dev/golang/src/github.com/anycable/anycable-go/node/hub.go:105 +0x1e5 
created by github.com/anycable/anycable-go/node.NewNode 
/Users/palkan/dev/golang/src/github.com/anycable/anycable-go/node/node.go:90 +0x14c 
```

**AnyCable-Go version:** 0.6.0-preview4
**AnyCable gem version:** 0.5.0

**Comment by ernsheong** [comment]
I am also seeing this. 🙏

**Comment by palkan** [comment]
Fixed in `0.6.0-preview5` (see [release](https://github.com/anycable/anycable-go/releases/tag/v0.6.0-preview5))
